sfn-query-large-datasets-cdk/example-workflow.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Is there a testing or preview environemnt to see how this will look in Serverlessland before publishing?

Reply:
Not that I'm aware of. But I've aligned the content with the published ones.